lang/cs/Org.Apache.REEF.IO/FileSystem/AzureBlob/AzureCloudBlobClient.cs

```diff
@@ -62,6 +63,17 @@ public ICloudBlobContainer GetContainerReference(string containerName)
             return new AzureCloudBlobContainer(_client.GetContainerReference(containerName));
         }
 
+        public bool CreateContainerIfNotExists(Uri uri)
+        {
+            var uriSplit = uri.AbsolutePath.Split(new[] { "/" }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
See comment on interface about this function signature. Please move container name extraction to it's own function. Also, Uri has the property "Segments" so you shouldn't have to perform your own split.

Reply:
Using segments

Reply:
removed